src/graphql/main/schema.graphql

```diff
@@ -81,6 +81,12 @@ type AccountUpdateDisplayCurrencyPayload {
   errors: [Error!]!
 }
 
+interface AnyPrice {
```

Comment:
could `PriceInterface` be a better name than `AnyPrice`? `AnyPrice` make me think about something where the buyer can decide what he want to spend... maybe that's just how I feel about it but wondering if interface doesn't convey better the intent?

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/2298/commits/a0abbc6b716ae40839384a059442fbf5909f4abf